app/src/main/java/com/example/mydigitalcloset/SuccessPage.java

```diff
@@ -50,6 +52,10 @@ public void onClick(View view) {
             public void onClick(View view) {
                 Intent intent = new Intent(getApplicationContext(), AboutUsPageActivity.class); //change depending on romans pages
                 startActivity(intent);
+                Log.i("MyDigitalCloset", "You have successfully opened the About Us Page");
+                //Shows summited on the bottom of the screen
+                Toast.makeText(getApplicationContext(), "Welcome to the About Us Page!", Toast.LENGTH_SHORT)
+                        .show();
```

Comment:
good toasts

Reply:
Thank you for noticing Im trying to add as many toasts as I can to make the app look nicer.